dali/CMakeLists.txt

```diff
@@ -88,6 +83,7 @@ target_link_libraries(${dali_lib} PRIVATE "-Wl,--exclude-libs,${exclude_libs}")
 # Link in dali's dependencies
 message(STATUS "Adding dependencies to ${dali_lib}: '${DALI_LIBS}'")
 target_link_libraries(${dali_lib} PRIVATE ${DALI_LIBS})
+target_link_libraries(${dali_lib} INTERFACE fmt::fmt)
```

Comment:
It's certainly not `INTERFACE`. `INTERFACE` means: "the target is not using the library itself, but the client will not be able to use the target without linking against that library".
We could even go for `PRIVATE` and instruct the user to link against `libfmt` if they need to use it in their code. Client code may even use `{fmt}` in header-only mode and need not link `libfmt` at all.